middleware/publishable.js

```diff
@@ -34,6 +35,9 @@ module.exports = function *publishable(next) {
     return;
   }
 
+  // See more at npm/npm#9616 and cnpmjs/cnpmjs.org#1024
+  common.decodeScopedPackageName(this.params);
```

Comment:
Well, the best solution will be wait for npm fix their bug.

However, our private registry doesn't work before that.

This is a ugly fix, so never mind if think it is ugly.